<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BandPassIdealFreqFilterForm</class>
 <widget class="QWidget" name="BandPassIdealFreqFilterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QRadioButton" name="inputReject">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>102</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Rejeita Faixa</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Este é um filtro de passa/rejeita faixa no domínio da frequência.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelWarning">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>248</red>
         <green>8</green>
         <blue>31</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>248</red>
         <green>8</green>
         <blue>31</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>148</red>
         <green>148</green>
         <blue>148</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Atenção: esta implementação não é eficiente, portanto, use imagens de baixa resolução. Em breve providenciaremos uma solução.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>202</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="labelCutOff">
      <property name="text">
       <string>Frequência de Corte</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="inputCutOffFreq"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="labelFreqBand">
      <property name="text">
       <string>Tamanho da Faixa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="inputFreqBand"/>
    </item>
   </layout>
  </widget>
  <widget class="QRadioButton" name="inputShowSpectrum">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Apenas Mostrar Espectro</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
